config/software/rq.rb

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright 2016 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name "rq"
+default_version "0.9.2"
```

Comment:
Actually, why not [0.10.4](https://github.com/dflemstr/rq/releases/tag/v0.10.4)?

Reply:
I was keeping it to the existing one but we can totally do that.